.github/workflows/main.yml

```diff
@@ -3,43 +3,75 @@ on:
     branches:
       - master
       - stable
-name: Build
+      - actions-test
+  pull_request:
+    branches:
+      - master
+name: Linux CI
 jobs:
   build:
-    runs-on: macos-latest
-    name: Build and deploy
-    strategy:
-      fail-fast: false
-      matrix: # Build debug and production 
-        debug: ['', 'debug']      # '' if production, 'debug' for debug
+    runs-on: ubuntu-latest
+    name: Linux build & test
     env:
-        BINTRAY_API_KEY_DEFINED: ${{!!secrets.BINTRAY_API_KEY}}
+      PROCESSORS: 0
+      PREFIX: "/tmp/ADALIB_DIR"
+      GPR_PROJECT_PATH: /tmp/ADALIB_DIR/share/gpr
     steps:
       - name: Get libadalang
         uses: actions/checkout@v2
+
+      - name: Show PR branch ref
+        run: |
+            echo ${{ github.head_ref }}
+            echo ${{ github.repository_owner }}
+            echo ${{ github.event.pull_request.head.label }}
```

Comment:
Is this still relevant? (now that you managed to get the `gh-langkit-branch.py` script working)